clients/pyethereum:develop/pyethapp.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Immediately abort the script on any error encountered
+set -e
+
+if [ $HIVE_TESTNET ];
+then
+  # TODO: Write better sed magic?
+  sed -i "s/  ACCOUNT_INITIAL_NONCE.*/  ACCOUNT_INITIAL_NONCE: 1048576/g" /root/.config/pyethapp/config.yaml
```

Comment:
Funnily enough, that test is the _one_ smoke test that pyethereum doesn't pass. (due to a different interpretation of getTransactionCount)